load-test/README.md

```diff
@@ -0,0 +1,71 @@
+# Run an Iter8 Load Test Experiment
+
+This action runs an Iter8 [load test experiment](https://iter8.tools/0.8/tutorials/load-test/overview/). For more details about Iter8 experiments, see <https://iter8.tools>.
+
+## Usage
+
+Load test and validate the the HTTP service whose URL is <https://example.com>. We will specify that the error rate must be 0, the mean latency must be under 50 msec, the 90th percentile latency must be under 100 msec, and the 97.5th percentile latency must be under 200 msec.
+
+``` yaml
+- uses: iter8-tools/load-test@v1
+  with:
+    url: http://example.com
+    error_rate: 0
+    mean_latency: 50
+    p90: 100
+    p97_5: 200
+```
+
+Set the number of requests sent during the load-test to 200, the number of requests per second to 10, and the number of parallel connections used to send the requests to 5:
+
+``` yaml
+- uses: iter8-tools/load-test@v1
```

Comment:
How/where do you set @v1 in the action? 

Also, what is the way to set it as `v0.8` instead of `v1`? I have seen many GH actions to `v1`, `v2`, ... perhaps, this is not intended to be a semver tag?

Reply:
Also... if there's a breaking change to Iter8 CLI (which will cascade into the Iter8 image also).

What happens to pipelines which currently use this action?
***

Less obvious ... if there's a breaking change to the experiment chart...

What happens to pipelines which currently use this action?
***
The above two questions are part of the Iter8-wide semver choices we need to make.

Reply:
> How/where do you set @v1 in the action?
> 
> Also, what is the way to set it as `v0.8` instead of `v1`? I have seen many GH actions using `v1`, `v2`, ... perhaps, this is not intended to be a semver tag?

The value placed after the `@` is a github reference. If you use `@main` it uses the main branch.  If you use `@pull/1150/head` it will use this pull request.  If you use `@v0.8.x` it will use that release.

For clarity (and desire not to change frequently), I think it desirable to document it as `@v0.8`. However, I am not sure that this works from a versioning perspective.  If we use `@v0.8.x` I think any issues of incompatibility go away.

Reply:
Creating the action in a separate project would allow for semantic versioning that does not exactly match the version of iter8; ie, `v0.8`. However, this introduces (a) a new repo and (b) the semantic versioning questions.